src/type/definition.js

```diff
@@ -563,26 +566,6 @@ export class GraphQLScalarType {
     }
   }
 
-  // Serializes an internal value to include in a response.
-  serialize(value: mixed): mixed {
-    const serializer = this._scalarConfig.serialize;
```

Comment:
>  it seems like an improvement based purely on the fact that the flow types are better

@mjmahone This wrapper function depends on `this` so before proposed change you can't do:
```js
const serialize = GraphQLString.serialize;
serialize(5);
// TypeError: Cannot read property '_scalarConfig' of undefined
```
Or more complicated scenario:
```js